tests/test_attacking_component.py

```diff
@@ -2,11 +2,11 @@
 from gym.spaces import MultiBinary
 import numpy as np
 
-from admiral.envs.components.agent import AttackingAgent
+from admiral.envs.components.agent import AttackingAgent, ComponentAgent
 from admiral.envs.components.actor import AttackActor
 
 
-class AttackTestAgent(AttackingAgent):
+class AttackTestAgent(AttackingAgent, ComponentAgent):
```

Comment:
Why not make all component agents inherit from ComponentAgent since they all assume that agents in the environment will have those attributes?